Assets/MixedRealityToolkit/Utilities/ClippingPrimitive.cs

```diff
@@ -117,6 +168,11 @@ protected void LateUpdate()
             UpdateRenderers();
```

Comment:
Should be wrap this UpdateRenderers call in a "if (useOnPreRender)" too?

Reply:
Its not necessary as `OnCameraPreRender(CameraEventRouter router)` will never get called if `UseOnPreRender` wasn't set.

Reply:
Per Desk conversation, I agree. Its deferring LateUpdate's call to OnPreRender.